homeassistant/components/wemo/entity.py

```diff
@@ -170,19 +185,38 @@ async def async_added_to_hass(self) -> None:
         registry = self.hass.data[WEMO_DOMAIN]["registry"]
         await self.hass.async_add_executor_job(registry.register, self.wemo)
         registry.on(self.wemo, None, self._subscription_callback)
+        if self.wemo.supports_long_press():
+            try:
+                await self.hass.async_add_executor_job(
+                    self.wemo.ensure_long_press_virtual_device
+                )
+            except (AttributeError, ActionException):
```

Comment:
Why can there be an attribute error? That's usually a sign of incorrect design. Attributes should be defined on instance init.

Reply:
Good catch. The code for this PR got out-of-sync with the rest of the component. Updated this similar to #46674 & #46705.